conference.js

```diff
@@ -2148,5 +2148,15 @@ export default {
             room.setDisplayName(formattedNickname);
             APP.UI.changeDisplayName(this.getMyUserId(), formattedNickname);
         }
+    },
+
+    /**
+     * Returns the desktop sharing source id or null if the desktop sharing is
+     * not active at the moment.
+     *
+     * @returns {string} - The source id.
```

Comment:
Should probably update the doc to signify null can be returned (or would it be undefined that's returned?).